src/components/common/CompoundLayout/components/Root.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, useColorMode } from "@chakra-ui/react"
+import { PropsWithChildren } from "react"
+import { LAYOUT_MAX_WIDTH_CSS_VAR } from "../constants"
+
+type Props = PropsWithChildren<{
+  background?: string
+  maxWidth?: string
+}>
+
+export function Root({
+  children,
+  background,
+  maxWidth = "sizes.container.lg",
+}: Props) {
+  const { colorMode } = useColorMode()
```

Comment:
We could clean up the code a bit by using the `useColorModeValue` hook here. I know we used `useColorMode` before this refactor too, but I think we could change it in this PR.